sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -309,6 +309,41 @@ You should see output similar to:
     node.js version     :  v6.10.3
     npm version         :  3.10.10
 
+The next check that can be made is to generate one of the example figures.
+There are a examples for the different ways in which bokeh can be used.
```

Comment:
I'd suggest "generate one of the example figures" -> "run some of the examples" and "There are a examples for the different ways" -> "There are different ways"